packages/valaxy/node/options.ts

```diff
@@ -48,10 +47,13 @@ export interface ValaxyPluginOptions {
     excerpt?: string
     path: string
   }) => void
+  plugins?: ValaxyPluginOption[]
 }
-
-export interface ValaxyThemePlugin extends ValaxyPluginOptions {
-  vite?: Omit<UserConfig, 'valaxy'>
+export type ValaxyPluginLike = ValaxyPlugin | ValaxyPlugin[] | false | null | undefined
+export type ValaxyPluginOption = ValaxyPluginLike | string | [string, any]
+export interface ValaxyPlugin extends ValaxyConfig {
+  enhanceAppFiles?: string[]
```

Comment:
We do not need `enhanceAppFIles` & `globalUIComponents`.
I think we should just use `setup/main.ts` & `components` folder.

Reply:
I think globalUIComponents should be kept as its way to hang on global components

As for `setup/main.ts` | `components`, I'm not very comfortable with it, it's great for theme development and user usage, but somewhat counter-intuitive in how plugins behave